### ISSUE TITLE
License missing

### ISSUE THREAD

**Issue by curiousleo** [seikichi/restructured#10]
Hi! I'm thinking of building a ReST + Firebase collaborative editor for fun - something like socrates.io but with ReST instead of Markdown.

It's just a fun little project, it would be open source and I have zero intention of commercialising it. Still, it would be good to know that you're okay with people playing with your code (and the "live demo" example) - so please consider adding a license to this project.

Thanks!

**Comment by lextm** [comment]
The readme file does say MIT.

**Comment by cco3** [comment]
True, but three letters don't exactly constitute a license.  It would be good to have an actual LICENSE file.
